template/src/components/app.tsx

```diff
@@ -1,23 +1,21 @@
-import { FunctionalComponent, h } from 'preact';
```

Comment:
AFAIK, using `FunctionalComponent` (or whatever the equivalent is in React, maybe `FunctionComponent`? I forget) has become a bit of an anti-pattern and generally not useful anyways.